loopchain/blockchain/blocks/block_versioner.py

```diff
@@ -37,6 +38,13 @@ def get_height(self, block_dumped: Union[str, dict]):
         height = block_dumped["height"]
         return int(height, 16) if isinstance(height, str) else height
 
+    def get_hash(self, block_dumped: Union[str, dict]):
+        if isinstance(block_dumped, str):
+            block_dumped = json.loads(block_dumped)
+
+        _hash = block_dumped["block_hash"]
```

Comment:
1. It is not compatible with block version 0.3. It has "hash"
2. Appending single underscore is better to avoid conflicting reversed words. Or just use 'block_hash'